ansible/vars/topo_t2.yml

```diff
@@ -0,0 +1,1867 @@
+topology:
+  # 4 DUTs - 3 line cards (dut 0, 1, 2) and 1 Supervisor card (dut 3) -  24 VMs per linecard
+  # line card dut0 is connected to 24 T3s (8 connections as 2-port LAGs and the rest as regular ports)
+  # line card dut1 and dut2 are connected to 24 T1s each (8 connections as 2-port LAGs and the rest as regular ports)
+  # ptf ports are numbered 0-95.
+  dut_num: 4
+  VMs:
+    ARISTA01T3:
+      vlans:
+        - 0.0@0
+        - 0.1@1
+      vm_offset: 0
+    ARISTA03T3:
+      vlans:
+        - 0.2@2
+        - 0.3@3
+      vm_offset: 2
```

Comment:
The vm_offset is used to map the VM in topology to server VM names, like (ARISTA01T3 -> VM0300).
If there is gap in vm_offset number, then ARISTA03T3 would be mapped to VM0302. Then VM0301 would not be used.
Suggest to make the vm_offset continuous without gap.

Reply:
Agreed - will make the change once we get a resolution on the 20.0.0.x vs 10.0.0.x being used.

Reply:
Committed the changes